pkg/cdc/generate_config.go

```diff
@@ -90,17 +91,46 @@ func updateAdditionalJVMOpts(optsSlice []string, CDCConfig *cassdcapi.CDCConfigu
 			optsSlice = append(optsSlice, nameTag+"="+fmt.Sprintf("%s", reflectedValue))
 		}
 	}
+	// We need to add these two elements to the slice first, because the management agent must start before the CDC agent.
```

Comment:
Is this comment misplaced? It looks like a duplicate of the one on line 103.

Reply:
Probably, this is why some people say don't write comments - they fall out of date :/ 

I'm not sure what my thinking was there so I'm getting rid of these.